vllm/sequence.py

```diff
@@ -256,7 +257,7 @@ def output_token_ids(self) -> Tuple[int, ...]:
         return tuple(self._output_token_ids)
 
     @output_token_ids.setter
-    def output_token_ids(self, new_output_token_ids: List[int]) -> None:
+    def output_token_ids(self, new_output_token_ids: Tuple[int, ...]) -> None:
```

Comment:
In `test_sampler.py`, a list is actually being assigned. Perhaps we should use a type annotation of `typing.Sequence[int]` to cover both cases.